Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1908,6 +1909,9 @@ def gen2018HiMix(fragment,howMuch):
 steps['RECODR2_2018reHLTAlCaTkCosmics_Prompt']=merge([{'--conditions':'auto:run2_data_promptlike'},steps['RECODR2_2018reHLTAlCaTkCosmics']])
 
 steps['RECODR2_2018reHLT_Prompt_hBStar']=merge([{'--era':'Run2_2018_highBetaStar'},steps['RECODR2_2018reHLT_Prompt']])
+steps['RECODR2_2018reHLT_skimJetHT_Prompt_HEfail']=merge([{'--conditions':'103X_dataRun2_Prompt_v1HEfail_v1'},steps['RECODR2_2018reHLT_skimJetHT']])
```

Comment:
https://github.com/cms-sw/cmssw/pull/24424/commits/428c71ffced73d476582273bc57a711d0732ad49#r214294240

Reply:
@franzoni What about adding two autoCond dictionaries for HE fail data and MC GTs?

Reply:
HI! Yes this was the plan suggested by Giovanni.  We can prepare a commit to be added in this PR.